graph-db/src/utils/cmr/harvestCmrCollections.js

```diff
@@ -0,0 +1,31 @@
+const { fetchPageFromCMR } = require('./fetchPageFromCmr');
+const { clearScrollSession } = require('./clearScrollSession');
+
+/**
+ * Harvest UMM JSON collection metadata from CMR environment set in env variable
+ * @returns {JSON} partitioned array of CMR collection search results
+ */
+exports.harvestCmrCollections = async () => {
+  const response = await fetchPageFromCMR();
+  const results = (await response.json()).items;
+  const scrollId = response.headers.get('CMR-Scroll-Id');
+  const partitionedSearchResults = [];
+  let continueScroll = true;
+
+  partitionedSearchResults.push(results);
+  while (continueScroll) {
+    const scrolledResults = await fetchPageFromCMR(scrollId).then((results) => results.json());
+    partitionedSearchResults.push(scrolledResults.items);
+
+    console.log(`Got [${scrolledResults.items.length}] items from the CMR`);
+    if (scrolledResults.items.length < process.env.PAGE_SIZE) {
```

Comment:
What if the last page is the same as `PAGE_SIZE`?

Reply:
The next page would have 0 results